api/v3/openapi.yaml

```diff
@@ -1610,6 +1612,8 @@ components:
                   - SET
                   - DELETE
                 nullable: true
+        validate_only:
```

Comment:
Reminder that you'll need to update the both the cc and cp consolidated-openapi.yaml in ce-kafka-rest for the CLI to be able to use these changes

Reply:
Thank you, filed KREST-8552.